e2e_samples/parking_sensors/scripts/deploy_infrastructure.sh

```diff
@@ -178,18 +178,25 @@ az keyvault secret set --vault-name "$kv_name" --name "applicationInsightsConnec
 # ###########################
 echo "validate_password function..."
 # ###########################
-###if there is an hyphen in the first position replace by R
+###if there is an hyphen reset until valid password in found in case of error empty the pwd.
 # ###########################
+
 validate_password() {
```

Comment:
I would suggest moving this function at the beginning of the script for better code readability

Reply:
done